mailet/standard/src/main/java/org/apache/james/transport/mailets/AmqpForwardAttribute.java

```diff
@@ -121,21 +124,24 @@ public void service(Mail mail) throws MailetException {
         }
     }
 
-    @SuppressWarnings("unchecked")
-    private Stream<byte[]> getAttributeContent(Mail mail) throws MailetException {
-        Serializable attributeContent = mail.getAttribute(attribute);
+    private Stream<byte[]> getAttributeContent(Attribute attribute) throws MailetException {
+        return extractAttributeValueContent(attribute.getValue().value())
```

Comment:
To get the raw value I saw before as well in `AttributeUtils` using `attribute.getValue().getValue()`... And checking in `AttributeValue` I can see indeed two getters (`value` and `getValue`) that return the same attribute. 

I think we can agree it's unnecessary and maybe do a quick refactoring? (but better in an other PR I guess, this one being big enough already :) )